Dockerfile

```
@@ -13,36 +13,39 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-## creating building container
-FROM python:3.12-slim-bookworm AS builder
-# update and install dependencies
-RUN apt update
-RUN apt upgrade -y
+# BASE: a base image with updated packages
+FROM python:3.12-alpine AS base
+RUN apk update
+RUN apk upgrade --available
```

```suggestion
RUN apk upgrade --no-cache --available
```

reduces image size and spares one layer